### DISCUSSION TITLE
Model calibration + confidence learning?

### DISCUSSION THREAD

**Discussion by andreimargeloiu** [cleanlab/cleanlab#1035] [Q&A]
I couldn't find any discussions on whether it is necessary or helpful to perform model calibration when identifying noisy labels in confidence learning (CL). CL utilizes the model's probabilities in combination with ranking, so it seems intuitive that a calibrated model might improve the detection of noisy labels. 

Does anyone have insights on this topic?

**Comment by fkroeber** [comment]
There was a corresponding issue on this: https://github.com/cleanlab/cleanlab/issues/75. Summary: It is not necessary to calibrate. Still, some empirical comparisons between using calibrated and uncalibrated probabilities would be nice!
